common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1408,3 +1417,30 @@ def clean_id(self, padding_char='='):
         return "course_{}".format(
             b32encode(unicode(self.location.course_key)).replace('=', padding_char)
         )
+
+    @property
+    def teams_enabled(self):
+        """
+        Returns whether or not teams has been enabled for this course.
+        """
+        if self.teams_configuration:
+            return 'topics' in self.teams_configuration and len(self.teams_configuration['topics']) > 0
```

Comment:
Nit: could be shorter - `return len(self.teams_configuration.get('topics', []))`.